tests/km_core_tests.bats

```diff
@@ -1860,8 +1868,8 @@ EOF
    run touch $WORKDIR/continue
```

Comment:
Do we need `run` here? Feels unnecessary

Reply:
If touch fails, the test will hang.

Reply:
why would it fail?

Reply:
I don't know.  It shouldn't fail but we are testing assumptions.
Maybe we could remove all the checks since the code shouldn't fail.